data/core/step1_get_started.md

```diff
@@ -149,12 +175,13 @@
     - slot{"onboarding": true}
     - utter_built_bot_before
 * deny
-    - utter_explain_stack
-    - utter_stack_details
-    - utter_explain_nlucore
-* affirm OR how_to_get_started{"product":"stack"} OR explain
+    - utter_explain_rasa_components
+    - utter_rasa_components_details
+    - utter_ask_explain_nlucorex
+* affirm OR how_to_get_started{"product":"all"} OR explain
```

Comment:
can we add OR explain to the others? looking at the NLU data it seems to make sense here